ferrocene/tools/traceability-matrix/src/annotations.rs

```diff
@@ -156,14 +154,12 @@ impl Annotations {
                     AnnotationSource::TestItself
                 } else if annotated_in_parent(annotation, file) {
                     AnnotationSource::ParentDirectory { bulk_file: shrink_path(&annotation.file) }
-                } else if annotation.file == file.file.join("Makefile") {
-                    AnnotationSource::Makefile
                 } else if annotation.file == file.file.join("rmake.rs") {
                     AnnotationSource::Rmake
                 } else {
                     anyhow::bail!(
-                        "bug: annotation {annotation:?} doesn't come from the file itself, \
-                        its parent directory, or a Makefile. \
+                        "bug: annotation {annotation:?} doesn't come from the file itself or \
```

Comment:
we should mention rmake file here, something that was missing previously

Reply:
Done